clients/src/main/java/org/apache/kafka/common/metrics/Metrics.java

```diff
@@ -171,6 +195,7 @@ public void removeSensor(String name) {
                     }
                 }
             }
```

Comment:
Is it worth recording this decision with the rationale somewhere for the benefit of other people reading the code?